implementations/rust/ockam/ockam_transport_tcp/src/listener.rs

```diff
@@ -82,7 +82,7 @@ mod test {
             let _connection = listener.accept().await.unwrap();
         }
     }
-    #[test]
+    // #[test] Breaking in CI, likely due to contention with other instances binding the port
```

Comment:
@robinbudd FYI - if you're working on a branch with this test enabled, you may get strange CI behavior. Since we do a lot of builds in parallel in Github Actions, we are hitting a port resource collision and it's causing this test to hang the container when attempting to listen

Reply:
@jared-s Sorry for intervention. May I suggest generating random IP (127.*.*.*) and port (e.g. 4***) to reduce chance of collision?

Reply:
Hiya @vlisivka ! Yes, I agree something like that should be done.